packages/web/components/LandingPage/Platform/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import { useModal, useAuth } from '../../../hooks';
+import { toast } from '../../Toast';
+
+import Button from '../Button';
+
+import { Container, TextContainer, Title, Subtitle } from './styles';
+
+const Platform = () => {
+	const router = useRouter();
+	const { user } = useAuth();
+	const { openModal } = useModal();
+
+	const handleButtonClick = (e) => {
+		e.preventDefault();
+		if (!user?.email) {
+			openModal('login');
+		} else {
+			toast.info('Você está sendo redireciondo para o início');
+			router.push('/');
+		}
+	};
+
+	return (
+		<Container>
+			<TextContainer>
+				<Title>A vitrine tecnológica mais completa do semiárido</Title>
+				<Subtitle>
+					Um ambiente digital interativo voltado a difundir as tecnologias demandadas e
+					ofertadas na resolução de problemas do semiárido brasileiro.
+				</Subtitle>
+				<Button onClick={handleButtonClick}>Acesse agora</Button>
```

Comment:
After logging in, everything looks the same. What do you think about redirecting the user to the home screen after login?

Reply:
@mateus4k sounds good! I'll work on this suggestion.